packages/deployments/contracts/contracts_forge/core/connext/facets/TokenFacet.t.sol

```diff
@@ -410,7 +410,7 @@ contract TokenFacetTest is TokenFacet, FacetHelper {
 
     vm.prank(_owner);
     // no error message given bc shouldnt be able to find function
-    vm.expectRevert();
+    vm.expectRevert(TokenFacet.TokenFacet__addAssetId_badMint.selector);
```

Comment:
@liu-zhipeng you did this right? should we remove above comment, since we're giving error message now?

Reply:
removed!